frontend/src/components/RecordsList.tsx

```diff
@@ -189,6 +190,9 @@ export default function RecordsList({
 
   return (
     <Container fluid>
+      <BreadCrumb currPageTitle={dataName} />
+      <Title text={dataName} />
```

Comment:
These previously live outside of `RecordsList` and in the (parent) page components `RequestsPage`, `PatientsPage`, etc.

I moved them here to (1) write less code as we get to remove the `samplesQueryParam` prop for this component and (2) match the new pattern of `SamplesList`.